docs/install.rst

```diff
@@ -197,54 +252,56 @@ GSISSH, but it requires (a) a MongoDB server and (b) a properly set-up
-the HPC cluster (see blue arrows in the figure above). In our experience,
-a small VM instance (e.g., Amazon AWS) works exceptionally well for this.
+        The MongoDB server is used to store and retrieve operational data during the
+        execution of an application using RADICAL-Pilot. The MongoDB server must
+        be reachable on **port 27017** from **both**, the host that runs the
+        Ensemble Toolkit application and the host that executes the MD tasks, i.e.,
+        the HPC cluster (see blue arrows in the figure above). In our experience,
+        a small VM instance (e.g., Amazon AWS) works exceptionally well for this.
 
-.. warning:: If you want to run your application on your laptop or private
-            workstation, but run your MD tasks on a remote HPC cluster,
-            installing MongoDB on your laptop or workstation won't work.
-            Your laptop or workstation usually does not have a public IP
-            address and is hidden behind a masked and firewalled home or office
-            network. This means that the components running on the HPC cluster
-            will not be able to access the MongoDB server.
+        .. warning:: If you want to run your application on your laptop or private
+                    workstation, but run your MD tasks on a remote HPC cluster,
+                    installing MongoDB on your laptop or workstation won't work.
+                    Your laptop or workstation usually does not have a public IP
+                    address and is hidden behind a masked and firewalled home or office
+                    network. This means that the components running on the HPC cluster
+                    will not be able to access the MongoDB server.
 
-A MongoDB server can support more than one user. In an environment where
-multiple users use Ensemble Toolkit, a single MongoDB server
-for all users / hosts is usually sufficient.
+        A MongoDB server can support more than one user. In an environment where
+        multiple users use Ensemble Toolkit, a single MongoDB server
+        for all users / hosts is usually sufficient.
 
-**Install your own MongoDB**
+        **Install your own MongoDB**
 
-Once you have identified a host that can serve as the new home for MongoDB,
-installation is straight forward. You can either install the MongoDB
-server package that is provided by most Linux distributions, or
-follow the installation instructions on the MongoDB website:
+        Once you have identified a host that can serve as the new home for MongoDB,
+        installation is straight forward. You can either install the MongoDB
+        server package that is provided by most Linux distributions, or
+        follow the installation instructions on the MongoDB website:
 
-http://docs.mongodb.org/manual/installation/
+        http://docs.mongodb.org/manual/installation/
 
-**MongoDB-as-a-Service**
+        **MongoDB-as-a-Service**
```

Comment:
Eliminate the whole `**MongoDB-as-a-Service**` 'section' as we do not support mlab anymore.